### ISSUE TITLE
Restore command parse string error

### ISSUE THREAD

**Issue by iushas** [apache/kvrocks#1894]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.

### Version

2.6.0

### Minimal reproduce step

RESTORE key:0000002597151 0 "\x00@\xc8VXKeHogKgJ=[5V9_X^b?48OKF2jGA<f:iR@50o7dS3JV4Q6L68lC[GTA]0DaMg?_oSmcS2^N1J?ELSX@CfKQ7cM5aea\\ngY8a3LGgNVa9eRA46XS8>7ABe1>Jl9O\\Rm\\7IS:g\\U5b]=`ePo^LXhl4e:4P[DQNA3cgn<RH4UcdI6<V9S78NViC;^omK:<<Ldel:=iDmLB\t\x00\x84A\xfc\x1e\x9eg\x0bj"

this command can execute succesfully in redis, but got error in kvrocks. the error is "(error) ERR unexpected EOF". 

### What did you expect to see?

can execute successfully like redis

### What did you see instead?

(error) ERR unexpected EOF

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by git-hulk** [comment]
Hi @iushas 

Thanks for your feedback. Can you paste the right values of this key, so that we can add it into test cases?

**Comment by iushas** [comment]
@git-hulk 
## redis 
```
127.0.0.1:6379> RESTORE key:0000002597151 0 "\x00@\xc8VXKeHogKgJ=[5V9_X^b?48OKF2jGA<f:iR@50o7dS3JV4Q6L68lC[GTA]0DaMg?_oSmcS2^N1J?ELSX@CfKQ7cM5aea\\ngY8a3LGgNVa9eRA46XS8>7ABe1>Jl9O\\Rm\\7IS:g\\U5b]=`ePo^LXhl4e:4P[DQNA3cgn<RH4UcdI6<V9S78NViC;^omK:<<Ldel:=iDmLB\t\x00\x84A\xfc\x1e\x9eg\x0bj"
OK


127.0.0.1:6379> get key:0000002597151
"VXKeHogKgJ=[5V9_X^b?48OKF2jGA<f:iR@50o7dS3JV4Q6L68lC[GTA]0DaMg?_oSmcS2^N1J?ELSX@CfKQ7cM5aea\\ngY8a3LGgNVa9eRA46XS8>7ABe1>Jl9O\\Rm\\7IS:g\\U5b]=`ePo^LXhl4e:4P[DQNA3cgn<RH4UcdI6<V9S78NViC;^omK:<<Ldel:=iDmLB"

```

## kvrocks 

```
127.0.0.1:6379> RESTORE key0000002597151 0 "\x00@\xc8VXKeHogKgJ=[5V9_X^b?48OKF2jGA<f:iR@50o7dS3JV4Q6L68lC[GTA]0DaMg?_oSmcS2^N1J?ELSX@CfKQ7cM5aea\\ngY8a3LGgNVa9eRA46XS8>7ABe1>Jl9O\\Rm\\7IS:g\\U5b]=`ePo^LXhl4e:4P[DQNA3cgn<RH4UcdI6<V9S78NViC;^omK:<<Ldel:=iDmLB\t\x00\x84A\xfc\x1e\x9eg\x0bj"
(error) ERR unexpected EOF

```

**Comment by iushas** [comment]
you can use redis-benchmark  to insert  some keys in redis, and   use redis-shake to migrate from redis to kvrocks。 some key value cant be support by kvrocks

redis-benchmark -a xxx -h xx -c 250 -d 200 -e -q -n 3000000 -r 1000000 -t set

**Comment by git-hulk** [comment]
@iushas I see, it's a bug which was fixed in #1798. You can have a try at the latest version.

**Comment by git-hulk** [comment]
Close this issue, and feel free to reopen if this issue is still living.
